examples/server/server.cpp

```diff
@@ -2,8 +2,6 @@
 #include "llama.h"
 #include "build-info.h"
 
-// single thread
-#define CPPHTTPLIB_THREAD_POOL_COUNT 1
```

Comment:
I am wondering how this server can be multithreaded now?

Reply:
I added a mutex around the llama operations so that the server can multithread and only invocations of llama serialize. This is important now that the server also serves static assets.

Reply:
OK, we need to get multithreaded in the future anyway.